t/e2e/node/manager_test.go

```diff
@@ -258,11 +251,11 @@ func (s *ManagerTestSuite) TestStartNodeWithUpstreamEnabled() {
 	nodeConfig.UpstreamConfig.Enabled = true
 	nodeConfig.UpstreamConfig.URL = networkURL
 
-	s.NoError(s.NodeManager.StartNode(nodeConfig))
-	s.True(s.NodeManager.IsNodeRunning())
+	s.NoError(s.StatusNode.Start(nodeConfig))
+	s.True(s.StatusNode.IsRunning())
 
 	time.Sleep(100 * time.Millisecond) //https://github.com/status-im/status-go/issues/429#issuecomment-339663163
-	s.NoError(s.NodeManager.StopNode())
+	s.NoError(s.StatusNode.Stop())
 }
 
 // TODO(adam): race conditions should be tested with -race flag and unit tests, if possible.
```

Comment:
@adambabik It's been 6 moths since this TODO was added, can we get rid of this code?